engine/src/main/scala/cromwell/jobstore/JobStoreReader.scala

```diff
@@ -0,0 +1,17 @@
+package cromwell.jobstore
+
+import akka.actor.{Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.core.JobKey
+
+class JobStoreReader extends Actor with ActorLogging {
+  override def receive = LoggingReceive {
+    case IsJobCompletedYet(jobKey) => sender ! JobNotCompleteYet(jobKey)
+    case unknownMessage => log.error(s"Unexpected message to JobStoreReader: $unknownMessage")
+  }
+}
+
+case class IsJobCompletedYet(jobKey: JobKey)
+
+case class JobAlreadyCompleted(jobKey: JobKey, jobResult: JobResult)
+case class JobNotCompleteYet(jobKey: JobKey)
```

Comment:
Yeah I like `JobComplete` and `JobNotComplete`.  `JobRunning` seems to imply a more specific knowledge of state that I don't think this code will know?